Sources/Extensions/CoreGraphics/CGVectorExtensions.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  CGVectorExtensions.swift
+//  SwifterSwift
+//
+//  Created by Robbie Moyer on 7/25/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if canImport(CoreGraphics)
+import CoreGraphics
+
+// MARK: - Properties
+public extension CGVector {
+
+    /// SwifterSwift: The angle of rotation (in radians) of the vector.
+    /// The range of the angle is -π to π; an angle of 0 points to the right.
+    public var angle: CGFloat {
+        return atan2(dy, dx)
+    }
+
+    /// SwifterSwift: The magnitude (or length) of the vector.
+    public var magnitude: CGFloat {
```

Comment:
Can you add the math formula or a link resource to the explanation of how it is calculated on the docs? Although it is simple I think is good to add :))

Reply:
@moyerr +1 👆
You can even use the comment inside the code and move it to the documentation.